request.go

```diff
@@ -74,6 +74,7 @@ func (r RequestError) Error() string {
 const (
 	requestUUIDHeaderParam           = "X-Request-Id"
 	requestRateLimitResetHeaderParam = "ratelimit-reset"
+	requestRetryAfteHeaderParam      = "Retry-After"
```

Comment:
Can you make these constants more uniform? Some headers are written lower case, some upper-case. Make this the same.

Also can you make the constants shorter? `requestRetryAfteHeaderParam` has not only a typo but could be just `retryAfter`. Same for the others.

Reply:
Actually, there is no problems if some headers are written in lower case, and some in upper case. The reason is "it is case insensitive" [Info](https://golang.org/pkg/net/http/#Header.Get)

Reply:
Anyway, I did update to maintain the uniformity.

Reply:
Thanks!